FiducialRegistrationWizard/Logic/vtkPointDistanceMatrix.cxx

```diff
@@ -0,0 +1,200 @@
+#include "vtkPointDistanceMatrix.h"
+
+#include <vtkDoubleArray.h>
+#include <vtkMath.h>
+#include <vtkObjectFactory.h> //for new() macro
+
+//----------------------------------------------------------------------------
+vtkStandardNewMacro( vtkPointDistanceMatrix );
+
+//------------------------------------------------------------------------------
+vtkPointDistanceMatrix::vtkPointDistanceMatrix()
+{
+  pointList1 = vtkSmartPointer< vtkPoints >::New();
+  pointList2 = vtkSmartPointer< vtkPoints >::New();
+  distanceMatrix = vtkSmartPointer< vtkDoubleArray >::New();
+  UpdateNeededFlag = 0;
```

Comment:
This sounds like a really good idea. I noticed that in other classes (vtkLookupTable, vtkImageData, etc...) that the variable is declared **not** as a pointer. Is there a particular reason you suggest declaring it as a pointer, or should I just follow the examples in vtk?